### PR TITLE
Added support for missing Instance-related endpoints

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Added support for the following Instance endpoints:
- POST linode/instances/$1/disks/$2/clone
- GET linode/instances/$1/nodebalancers
- POST linode/instances/$1/password
- GET linode/instances/$1/transfer/$2/$3

### Unit Tests
`make testunit`

### Integration Tests
- `make fixtures ARGS="-run TestInstance_Disk_Clone"`
- `make fixtures ARGS="-run TestInstance_NodeBalancers_List"`
- `make fixtures ARGS="-run TestInstance_ResetPassword"`
- `make fixtures ARGS="-run TestInstance_GetMonthlyTransfer"`
- `make fixtures ARGS="-run TestInstance_GetTransfer"`

### CODE REVIEW

File: instance_nodebalancers.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package linodego
+
+import (
+	"context"
+)
+
+// ListInstanceNodeBalancers lists NodeBalancers
```
Comment (lgarber-akamai) [inline instance_nodebalancers.go:7]: Looks like this comment wasn't quite finished

Comment (ezilber-akamai) [reply instance_nodebalancers.go]: Oops. Good catch!

File: instance_disks.go [hunk 2]
```diff
@@ -165,3 +165,9 @@ func (c *Client) DeleteInstanceDisk(ctx context.Context, linodeID int, diskID in
 	err := doDELETERequest(ctx, c, e)
 	return err
 }
+
+// CloneInstanceDisk clones the given InstanceDisk for the given Instance
+func (c *Client) CloneInstanceDisk(ctx context.Context, linodeID, diskID int) (*InstanceDisk, error) {
```
[the remark this answers is not in the document]
Comment (ezilber-akamai) [reply instance_disks.go]: Good idea. Fixed!

**Review by jriddle-linode — APPROVED** [review]
LGTM, every thing is passing locally!

**Review by lgarber-akamai — APPROVED** [review]
Tests are passing on my end, great work!

**Pull request merged**